video/out/vo_vaapi_wayland.c

```diff
@@ -266,10 +243,6 @@ static int preinit(struct vo *vo)
         p->display = NULL;
         goto fail;
     }
-    va_get_formats(p);
-    if (!p->image_formats)
-        goto fail;
-
     vo->hwdec_devs = hwdec_devices_create();
```

Comment:
Could you keep the newline above `vo->hwdec_devs...` too?

Reply:
no problemo